x-pack/plugins/monitoring/public/alerts/alert_form.test.tsx

```diff
@@ -243,6 +247,9 @@ describe('alert_form', () => {
                 setActionFrequencyProperty={(key: string, value: any, index: number) =>
                   (initialAlert.actions[index] = { ...initialAlert.actions[index], [key]: value })
                 }
+                setActionAlertsFilterProperty={(key: string, value: any, index: number) =>
```

Comment:
nit: can the `value` be really anything? Usually there is a general recommendation to use stricter types even in tests as it can help to catch some problems. `unknown` may be a better fit here.

Reply:
The behavior of these `setSomething` functions aren't being tested here, which is why we've set these type signatures, basically just to get typescript to pass. But I can change it to `unknown`